cmd/notation/inspect.go

```diff
@@ -71,8 +72,8 @@ Example - Inspect signatures on an OCI artifact identified by a digest and outpu
   notation inspect --output json <registry>/<repository>@<digest>
 `,
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) == 0 {
-				return errors.New("missing reference")
+			if err := arg.ValidateCount(args, 1, "missing reference"); err != nil {
```

Comment:
Can we use Cobra buit-in [Positional and Custom Arguments](https://github.com/spf13/cobra/blob/main/user_guide.md#positional-and-custom-arguments)?